src/helperFunctions.js

```diff
@@ -7,15 +7,24 @@ function writeToFile(filePath, txt) {
   return fs.writeFileSync(filePath, body);
 }
 
+function hasApiKey(api_key) {
+  if(api_key !== '' || typeof(api_key) !== 'undefined') {
+    return false
+  } else {
+    return true
+  }
+  
+}
+
 function getConfigFile(res) {
   let uri = res[0]['path'];
   return vscode.workspace
     .openTextDocument(uri)
     .then((doc) => {
       let text = doc.getText();
       text = JSON.parse(text);
```

Comment:
I know you didn't write this line, but maybe a new var name would make sense after parsing? Perhaps `config`?